Rajitha/Test2.js

```diff
@@ -0,0 +1,39 @@
+const isOpen = true;
+const isClosed = false;
+const myTitle = "The best show in the world";
+
+function isOn (isActive) {
+    return isActive
+};
+
+let isShowing = function ( isHidden )  {
+    return !isHidden;
+};
+
+let getProps = function ( title ) {
+    return { title: title}
+};
+
+let someValue = isOn(false) || getProps("yo"); //returns objects
+console.log("someValue1 ", +someValue);
+
+let someValue1 = isOn(isOpen) || getProps("yo") || {}; //returns true
+console.log("someValue1 ", +someValue1);
+
+let someValue2 = isOn(true) || getProps() || {}; //returns true
+console.log("someValue2 ", +someValue2);
+
+let someValue3 = isOn(isClosed) || isShowing(isClosed); //returns true
+console.log("someValue3 ", +someValue3);
+
+console.log("check this " + isShowing(false)); //return true
+console.log("****", +isOn(isClosed)); //why is it returning 0 instead of false
```

Comment:
you added a `+` sign in front of `isOn(isClosed))`.  Add a + to false gets a number 0, Adding a + to true gets a number 1. 

You're essentially converting true/false to a number. Its the opposite of the double bangs / nots.

!!0 = false
!!1 = True